strapi/strapi
# [DEITS] Add tests and clean up import/export

### What does it do?

Import command:

- adds a test
- fixes that we weren't using createStrapiInstance
- moves the logic for file extension checking to the CLI (CLI should own it so that user could have option to use decryption/decompression regardless of filename)

Export command:

- cleans up tests

### Why is it needed?

Tests are good, and this also fixes some minor issues with consistency between the commands.

### How to test it?

- All tests should pass

- import / export command should work exactly as before, using files that are:
  - encrypted and compressed
  - encrypted but not compressed
  - compressed but not encrypted)
